docker/.env-template

```diff
@@ -0,0 +1,5 @@
+POSTGRES_HOST="cms_db"
+POSTGRES_USER="cms"
+POSTGRES_PASSWORD="cms"
+POSTGRES_DB="cms"
+POSTGRES_PORT=54321
```

Comment:
this file can be called `local.env` and used as it for dev, it doesnt have to be a template